src/ui/component/copyToClipboard/copyToClipboard.scss

```diff
@@ -1,9 +1,48 @@
+@mixin hide-icons($icons) {
+  .okp4-dataverse-portal-copy-paste-icon {
```

Comment:
It's correct, but I'd love to have a check if `$icons` contains values from a specific list to prevent typos.

Reply:
That's a great suggestion! I've implemented it together with a error warning  👍